src/components/ae-input-range/ae-input-range.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <input
+    class="ae-input-range"
+    type="range"
+    :class="[fill]"
+    :value="value"
+    v-bind="$attrs"
+    @input="propagateEventValue"
+  />
+</template>
+<script>
+import { events } from '../../mixins';
+
+export default {
+  name: 'ae-input-range',
+  mixins: [events],
+  props: {
+    /**
+     * Select the fill/style that you want to apply
+     * for the range input element.
+     * Valid input: `primary, secondary, neutral, alternative`
+     */
+    fill: {
+      type: String,
+      validator: value => [
+        'primary',
+        'secondary',
+        'neutral',
+        'alternative',
+      ].includes(value),
+      default: 'primary',
+    },
+
+    /**
+     * Value property of the range input element
+     */
+    value: [String, Number],
```

Comment:
It concerned to be a good practice set a default value or mark property as required, why you have removed it from our version? https://github.com/aeternity/aepp-base/blob/ad418a623369031bf825325520dffc7c9e6a24a5/src/components/AeInputRange.vue#L15

Reply:
```
value
The input control's value. When specified in the HTML, this is the initial value, and from then on it can be altered or retrieved at any time using JavaScript to access HTMLInputElement.value. The value attribute is always optional.
```

The value is optional.